libcloud/httplib_ssl.py

```diff
@@ -162,9 +162,10 @@ class LibcloudConnection(LibcloudBaseConnection):
     host = None
     response = None
 
-    def __init__(self, host, port, **kwargs):
+    def __init__(self, host, port, secure=None, **kwargs):
+        scheme = 'https' if secure is not None and secure else 'http'
```

Comment:
I think the defaults would be more clear if this read:

```
def __init__(self, host, port, secure=None, **kwargs):
    if secure is None and port == 443:
        secure = True
    scheme = 'https' if secure else 'http'

    # ... rest of the code
```

I dislike inline logic because it is hard to find for the casual reader.

Also, a docstring would be nice.